docs/source/fractal-services.rst

```diff
@@ -0,0 +1,26 @@
+Services
+========
+
+Services are unique workflows where there is an iterative component on the server. A typical service workflow looks like the following:
+
+ 1. A client submits a new service request to the server.
+ 2. A service is created on the server and place in the service queue.
+ 3. A service iteration is called that will spawn new tasks.
+ 4. The service waits until all generated tasks are complete.
+ 5. The service repeats 3 and 4 until the service iterations are complete.
+ 6. The service cleans intermediate data, finalizes the data representation, and marks itself complete.
+
+To give an example, the TorsionDrive Service will be used. This service optimizes the geometry of a biomolecule at a number
+of frozen dihedral angles to provide an energy profile of the rotation of this dihedral bond. Here the geometry
```

Comment:
"The TorsionDriver Service is an example. The service"

Dangling "Here the geometry"